go/vt/sqlparser/parse_test.go

```diff
@@ -959,7 +955,7 @@ var (
 		input: "create table `by` (\n\t`by` char\n)",
 	}, {
 		input:  "create table if not exists a (\n\t`a` int\n)",
-		output: "create table a (\n\ta int\n)",
+		output: "create table if not exists a (\n\ta int\n)",
```

Comment:
(I think...) This was syntax we preserve after your previous changes but that upstream didn't originally.

Reply:
Yup, I added support for if not exists (always parsed, but wasn't part of the AST). Looks correct.